centreon/tests/e2e/features/Dashboards/13-dashboard-all-widgets.feature

```diff
@@ -0,0 +1,33 @@
+Feature: Configuring dashboard with all widgets
+  As a Centreon User with dashboard update rights,
+  I must be able to comprehensively configure all widgets within a dashboard
+  This includes manipulating the properties of each widget and thoroughly testing the outcomes of these manipulations
+
+  @TEST_MON-37042
+  Scenario: Adding all widgets into the same dashboard
+    Given the dashboard administrator redirected to dashboard interface
+    When the dashboard administrator add generic text widget
+    And the dashboard administrator add single metric widget
+    And the dashboard administrator add metrics graph widget
+    And the dashboard administrator add top bottom widget
+    And the dashboard administrator add Status grid widget
+    Then the dashboard administrator save the dashboard
```

Comment:
```suggestion
    And the dashboard administrator saves the dashboard
```

Reply:
should probably need another "Then" step like "then the dashboard is saved and in view mode" etc.